tests/integration_tests/test_game.py

```diff
@@ -409,6 +419,55 @@ async def test_game_ended_broadcasts_rating_update(lobby_server, channel):
         await asyncio.wait_for(mq_proto_all.read_message(), timeout=10)
 
 
+@fast_forward(30)
+async def test_double_host_message(lobby_server):
+    _, _, proto = await connect_and_sign_in(
+        ("test", "test_password"), lobby_server
+    )
+    await proto.send_message({
+        "command": "game_host",
+        "mod": "faf",
+        "visibility": "public",
+    })
+    await read_until_command(proto, "game_launch", timeout=10)
+
+    await proto.send_message({
+        "command": "game_host",
+        "mod": "faf",
+        "visibility": "public",
+    })
+    await read_until_command(proto, "game_launch", timeout=10)
+
+
+@fast_forward(30)
+async def test_double_join_message(lobby_server):
+    _, _, host_proto = await connect_and_sign_in(
+        ("test", "test_password"), lobby_server
+    )
+    guest_id, _, guest_proto = await connect_and_sign_in(
```

Comment:
Get rid of the unused guest_id then it looks good

Reply:
Aaargh. Copy/paste got me